### ISSUE TITLE
Get authorization code for scopes

### ISSUE THREAD

**Issue by ivsm** [thelinmichael/spotify-web-api-node#224]
I am trying to get an authorization token to get my own playlists but I don't know how to get the `authorizationCode = '<insert authorization code>';` as it is explained in the examples

Where can I get that code from?

**Comment by JMPerez** [comment]
@ivsm You are right. We could benefit from having some more complete examples on how to implement the authorization code flow with the wrapper.

You can have a look at [Spotidy's API quick start](https://developer.spotify.com/documentation/web-api/quick-start/) as well as the [GitHub repo](https://github.com/spotify/web-api-auth-examples/tree/master/authorization_code) that contains the example. There you can see how to obtain the authorization code. The wrapper functions would replace the requests made in that example.

Hope that helps!

**Comment by ivsm** [comment]
But I still don't know how to get that code. I want to use it in NodeJS without browser, is there any way to get it from backend(nodeJS) or manually from a website or something like that?

**Comment by JMPerez** [comment]
Since you need to access a user's playlists, even if its your own, you need to log in with your Spotify account and grant access to your app (eg client id). The only way to do it is through a browser.

Once you grant access, if you are implementing [the Authorization Code flow](https://developer.spotify.com/documentation/general/guides/authorization-guide/#authorization-code-flow), you will get a refresh token back. From that moment you can use that token to generate new access tokens when they expire. That step, [marked as step 4 in the flow documentation](https://developer.spotify.com/documentation/general/guides/authorization-guide/#4-requesting-a-refreshed-access-token-spotify-returns-a-new-access-token-to-your-app) consists of a POST request that you do directly from the backend.

So in short:
1) Implement authorization code flow
2) Log yourself in
3) Store access token and refresh token in the backend
4) Get a new access token when it expires using the refresh token

**Comment by JMPerez** [comment]
I'm closing the ticket since this issue is not related to the wrapper itself. For questions about spotify's authentication flows you can contact the team on Twitter at [@SpotifyPlatform](https://twitter.com/SpotifyPlatform).
